service/gcs/main.go

```diff
@@ -126,7 +131,7 @@ func main() {
 	if err := syscall.Sysinfo(&sinfo); err != nil {
```

Comment:
should the 75MB number in the comment above be increased to 110MB?

Reply:
Good catch the comment is now wrong.